scripts/revng-check-conventions

```diff
@@ -100,134 +109,244 @@ while [[ $# > 0 ]]; do
     esac
 done
 
+# Retrive files by extension or shebang
+# $1: extension to match
+# $2: shebang to match
+function find_by_ext_or_shebang() {
+    readarray -t ext_files < <($GIT_LS_FILES | grep "\.${1}$")
+    readarray -t dirty_files < <($GIT_LS_FILES)
+    shebang_files=()
+    for file in "${dirty_files[@]}"; do
+        if head -n1 "$file" | grep -qE "^#!.*${2}"; then
+            shebang_files+=("$file")
+        fi
+    done
+    files=( "${ext_files[@]}" "${shebang_files[@]}")
+    if [[ ${#files[@]} -eq 0 ]]; then return; fi
+    printf "%s\n" "${files[@]}" | sort -u
+}
+
+python_files="find_by_ext_or_shebang py python"
```

Comment:
This doesn't work with `python3`

Reply:
I've tried the following shebangs:
```
#!/usr/bin/env python3
#!/usr/bin/python3
```
and both are detected
Do you have a file to reproduce the issue?